packages/yoastseo/src/scoring/assessments/seo/ProductIdentifiersAssessment.js

```diff
@@ -57,6 +57,30 @@ export default class ProductIdentifiersAssessment extends Assessment {
 		return assessmentResult;
 	}
 
+	/**
+	 * Contains extra logic for the isApplicable method.
+	 *
+	 * @param {object} customData The custom data part of the Paper object.
+	 * @private
```

Comment:
I am not sure if adding @private in the JSDoc is necessary in this case. At least we haven't done this before in other assessments classes that we have.

Reply:
I'll follow this. I only added it because PhP storm suggested it.